packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -631,6 +632,18 @@ class IsolateManager extends Disposer {
     // We don't yet known the main isolate.
     _mainIsolate.value = null;
   }
+
+  Future<Isolate> getIsolateCached(IsolateRef isolateRef) {
```

Comment:
The provider page creates a bunch of EvalOnDartLibraries whcih is slow because the large Isolate objects got fetched from the server for each instance which is a waste.